miRge.pl

```diff
@@ -36,7 +36,7 @@
 my $version = "2.0";
 local $ENV{PATH} = "$ENV{PATH}:".$miRgePath;
 $miRgePath =~ s/\/[^\/]+\.pl/\//;
-my $refPath = $miRgePath."miRge.seqLibs/";
+my $refPath = $miRgePath."usr/local/miRge/miRge.seqLibs/";
```

Comment:
I corrected the issue with the tar.gz so that it should just expand to miRge.seqLibs

Should be upload to the server within 5-10 min